src/cpx_model.jl

```diff
@@ -224,6 +224,27 @@ function set_warm_start!(model::Model, indx::IVec, val::FVec, effortlevel::Integ
     end
 end
 
+change_mip_start!(model::CPLEX.Model, x::Vector{Float64}, effortlevel::Integer = CPX_MIPSTART_AUTO) = change_mip_start!(model, Cint[1:length(x);], x, effortlevel)
```

Comment:
We're trying to standardize our naming of the low-level functions to `c_api_xxx`. See, e.g., https://github.com/JuliaOpt/CPLEX.jl/blob/cb0c9226f7a49aff5a2358b0c82a3ebb5151a340/src/cpx_constrs.jl#L1.

So this should be named `c_api_chgmipstarts`.

Reply:
I did not know for this standardization, I read the pull request from 2017, corrected now.

Reply:
No worries. This wrapper is one of the older JuliaOpt packages. It could do with a big clean up.